src/IO/AbstractIOHandlerHelper.cpp

```diff
@@ -82,7 +85,10 @@ namespace openPMD
 #if openPMD_HAVE_ADIOS2
             case Format::ADIOS2:
                 return std::make_shared< ADIOS2IOHandler >(
-                    path, access, std::move( optionsJson ) );
+                    path, access, std::move( optionsJson ), "bp4" );
```

Comment:
What about a JSON parameter `.backend`? We can leave the filename extensions in as shortcuts, including a `bp4` one. We would then of course need to actually store those things as `*.bp4` on disk, otherwise this trick won't work when reading things back in.